README.md

```diff
@@ -1,2 +1,166 @@
-# React-Capstone
-This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux.
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 React Capstone - CoinGecko <a name="about-project"></a>
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux. It's connected to CoinGecko API to get several data about cryptocurrencies.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-react">REACT</a></li>
+    <li><a href="https://redux-toolkit.js.org/">REDUX TOOLKIT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **Use React components**
+- **Use React Router**
+- **Understand the concept of state management**
+- **Use store, actions, and reducers in React**
+- **Setup a Redux store**
+- **Create a slice to save state and reducers**
+- **Connect React and Redux**
+- **Connect an existing API via Redux and use the data to display something in a UI**
+- **Apply styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live site](https://coingecko-4xi2.onrender.com/metrics)
+- [Loom video](https://www.loom.com/share/334c5aca12bc46f48e7bd8a587e0dc19)
+
```

Comment:
- [ ] The live demo version of the app is not displaying data on the Frontend but just keeps loading forever. Kindly check it out so the hosted version of your app works fine as expected 👍🏽
- [ ] And lastly, I think something went wrong when you were recording your video presentation. There is no voice in the video. I'm sure the issue is from your mic when recording. Kindly make redo the video and check out the sound 👍🏽

<img width="1553" alt="Screenshot 2023-06-10 at 8 00 23 PM" src="https://github.com/CarlosIgreda/React-Capstone/assets/50941074/a0cbdda4-d655-45d4-9091-b6bc061e8dc2">

Reply:
Done it.